WeatherMonitor/Services/ConsoleOutputService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WeatherMonitor.Services.Interfaces;
+using WeatherMonitor.UI;
+
+namespace WeatherMonitor.Services;
+
+public class ConsoleOutputService : IOutputService
+{
+    public void Output(string message)
+    {
+        Console.WriteLine();
+        DisplayHelper.Print("[Bot Output]: ", ConsoleColor.DarkCyan);
+        DisplayHelper.PrintLine("message", ConsoleColor.DarkGreen);
```

Comment:
Bug- This functionality is not working properly .. maybe you need to utilise the parameter not hardcoded value
`$"message: {message}"`
![Screenshot_504](https://github.com/user-attachments/assets/6fe8cbe3-f292-4950-baba-9d0a0d84f252)

Once changed:
![image](https://github.com/user-attachments/assets/0c2cfb87-6a33-478c-afc5-cdc0a94cc533)